components/chat-form/chat-form.js

```diff
@@ -1,5 +1,83 @@
 (function () {
   'use strict';
 
+  // Some data needs to survive between executions of Drupal's `attach` method
+  // so we instantiate it outside the Behavior itself.
+  var oldScrollHeight;
+
+  // Initialize. We do this outside Drupal.behaviors because it doesn't need to
+  // run each time ajax gets called.
   window.parent.postMessage('ready', window.origin);
+
+  Drupal.behaviors.ochaAiChatForm = {
+    attach: function (context, settings) {
+      var chatContainer = document.querySelector('[data-drupal-selector="edit-chat"] .fieldset-wrapper');
+      var submitButton = document.querySelector('[data-drupal-selector="edit-submit"]');
+
+      // Add padding equal to chat window so we can always scroll. We take 16px
+      // away to avoid overages due to padding-bottom.
+      var chatHeight = chatContainer.getBoundingClientRect().height - 16;
+      chatContainer.style.paddingBlockStart = chatHeight + 'px';
```

Comment:
That results in a big blank space when scrolling up when there are some questions and answers. I wonder if that should not be recalculated when responses are received.

Reply:
Great point. I'll convert this to a CSS Var and update the value on window resize.

Reply:
Now I see what you mean. I guess we could recalculate when answers arrive to keep the scroll area as small as possible while forcing a scroll overflow large enough to facilitate scrolling. If it interferes with the scrolling effect, I'd tend to leave it alone; IMO mimicking a modern feel for the chat provides significant affordance and makes it feel more reliable.